deployments/charts/kuma/templates/cp-ingress.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if .Values.controlPlane.ingress.enabled }}
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ include "kuma.name" . }}-control-plane
+  namespace: {{ .Release.Namespace }}
+spec:
+  ingressClassName: {{ include "kuma.name" . }}-control-plane
+  rules:
+  - http:
+      paths:
+      - path: /
```

Comment:
I think we need more params here. For example kong has
```
  # Kong admin ingress settings. Useful if you want to expose the Admin
  # API of Kong outside the k8s cluster.
  ingress:
    # Enable/disable exposure using ingress.
    enabled: false
    ingressClassName:
    # TLS secret name.
    # tls: kong-admin.example.com-tls
    # Ingress hostname
    hostname:
    # Map of ingress annotations.
    annotations: {}
    # Ingress path.
    path: /
    # Each path in an Ingress is required to have a corresponding path type. (ImplementationSpecific/Exact/Prefix)
    pathType: ImplementationSpecific
```

Usually this ingress is a part of one gateway so we cannot just expose it on `*` hostname with `/` prefix.